tests/entities_column_value_simple_tests.py

```diff
@@ -101,4 +100,91 @@ def test_should_throw_exception_when_setting_an_invalid_value():
     column_value.value = text
 
     # Assert
-    eq_(column_value.value, str(text))
\ No newline at end of file
+    eq_(column_value.value, str(text))
+
+
+
+def test_should_create_a_column_value_with_no_api_input_data():
+
+    # Arrange
+    id = 'value_1'
+    title = "value"
+    column_type = ColumnType.numbers
+    column_value = en.create_column_value(column_type, id=id, title=title)
+
+    # Act
+    format = column_value.format()
+
+    #Assert
+    eq_(format, "")
+
+def test_should_create_a_column_value_with_api_input_data():
+
+    id = 'value_1'
+    title = "value"
+    column_type = ColumnType.numbers
+    value = "123"
+    column_value = en.create_column_value(column_type, id=id, title=title,value=value)
+
+    # Act
+    format = column_value.format()
+
+    #Assert
+    eq_(format, value)
+
+def test_should_setting_none_to_value():
+
+    # Arrange
+    id = 'value_1'
+    title = "value"
+    column_type = ColumnType.numbers
+    value=None
+    column_value = en.create_column_value(column_type, id=id, title=title)
+
+
+    # Act
+    column_value.value = value
+    format = column_value.format()
+
+    #Assert
+    eq_(value, None)
+
+def test_should_setting_an_int_or_float_to_value():
+    id = 'value_1'
+    title = "value"
+    column_type = ColumnType.numbers
+    value = 123.32
+    column_value = en.create_column_value(column_type, id=id, title=title)
+
+    # Act
+    column_value.value = value
+    format = column_value.format()
+
+    #Assert
+    eq_(float(format), value)
```

Comment:
Instead of the format (which should be removed), it would be better to test that _column_value.value_ equals _value_.